Editor/Scripts/Utilities/MeshsyncAnalytics.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+using UnityEditor;
+
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor.Analytics {
+
+    internal static class MeshSyncAnalyticsFactory {
+
+        public static IMeshSyncAnalytics CreateAnalytics() {
+            return new MeshSyncAnalytics();
+        }
+    }
+
+    internal interface IMeshSyncAnalytics {
+
+        /// <summary>
+        /// Send an event that a user installed and configured a DCC tool for use with MeshSync
+        /// </summary>
+        /// <param name="dccTool">Short designation for the DCC which was installed and configured</param>
+        void UserInstalledPlugin(string dccTool);
```

Comment:
Not necessarily blocking, but this could probably be an enum so the caller doesn't need to know the exact string that needs to be passed in

Reply:
I used a string specifically to avoid the extensibility problems caused by enums, the calling code is free to use enums and ToString them.